### DISCUSSION TITLE
Is there a collaboration feature available in the tool?

### DISCUSSION THREAD

**Discussion by normancarcamo** [Kong/insomnia#4123] [Help] [answered]
Hello there, I was looking into this tool and our team is looking for a tool with a subscription plan or self hosted solution which also includes team collaboration features such as comments, mentions/tagging pals etc.

Does anybody know if a feature like this is already on insomnia?

**Comment by wdawson** [comment, marked as answer]
We don't have those collaboration features (yet) in Insomnia. But we're thinking about what more we can do here. If you have specific ideas, feel free to add an Idea discussion and we can get community feedback there. 

Overall, we definitely want to improve on collaboration. We think that developing APIs with your team and working together can improve a lot about how productive we are and also the quality of the APIs we produce. This definitely falls within the scope of Insomnia's vision.
